src/test/with-feature-toggles.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { describe } from 'riteway';
+import dom from 'cheerio';
+import { withFeatureToggles } from '../with-feature-toggles';
+
+import { Consumer } from '../context';
+
+const render = ReactDOMServer.renderToStaticMarkup;
+
+const createTestComponent = () => ({ propCheck } = {}) => (
+  <Consumer propCheck={propCheck}>
+    {features => {
+      return (
+        <div>
+          <div className="component">{features.toString()}</div>
```

Comment:
cool, we can probably rename the class name to something that makes a bit more sense now, like maybe features